### PR TITLE
Fusion3-common, Dogo, Odin, Yuga: Update software binaries for AOSP KitKat (Android 4.4.4)

### PR DESCRIPTION
**Pull request by ghost**
From http://developer.sonymobile.com/downloads/tool/software-binaries-for-xperia-z2-xperia-z1-and-xperia-z/

### CODE REVIEW

**Comment by championswimmer** [comment]
+1

**Comment by chirayudesai** [comment]
s/for/to/ ;)

Also needs ramdisk updates, and other device configuration updates as needed.

**Comment by ghost** [comment]
What should I add to make these blobs work?

File: dogo/dogo-vendor-blobs.mk [hunk 1]
```diff
@@ -36,9 +36,12 @@ PRODUCT_COPY_FILES += \
     vendor/sony/dogo/proprietary/etc/firmware/touch_module_id_0x3f.img:system/etc/firmware/touch_module_id_0x3f.img \
     vendor/sony/dogo/proprietary/etc/flashled_calc_parameters.cfg:system/etc/flashled_calc_parameters.cfg \
     vendor/sony/dogo/proprietary/lib/hw/nfc.msm8960.so:system/lib/hw/nfc.msm8960.so \
+    vendor/sony/dogo/proprietary/lib/libOmxVenc.so:system/lib/libOmxVenc.so \
     vendor/sony/dogo/proprietary/lib/libaudcal.so:system/lib/libaudcal.so \
     vendor/sony/dogo/proprietary/lib/libcald_server.so:system/lib/libcald_server.so \
     vendor/sony/dogo/proprietary/lib/liboemcamera.so:system/lib/liboemcamera.so \
+    vendor/sony/dogo/proprietary/lib/modules/scsi_wait_scan.ko:system/lib/modules/scsi_wait_scan.ko \
+    vendor/sony/dogo/proprietary/lib/modules/wlan.ko:system/lib/modules/wlan.ko \
```
Comment (Incredible-O) [inline dogo/dogo-vendor-blobs.mk:44]: Meh :P

**Pull request closed without merging**